### ISSUE TITLE
Add support for datetime.datetime columns

### ISSUE THREAD

**Issue by Baruch4413** [koxudaxi/py-data-api#40]
**Is your feature request related to a problem? Please describe.**
I'm testing this driver, and so far it looks good, except for one thing.

I get this error when trying to insert a record:
Exception: unsupported type <class 'datetime.datetime'>: 2019-11-15 18:49:39.737104

**Describe the solution you'd like**
I don't know if I am doing something wrong, but it seems that this datatype is not yet supported.

**Additional context**
I am using Flask-SQLAlchemy==2.4.1

Thanks for your work on this package, I really apreciate it.

**Comment by koxudaxi** [comment]
@Baruch4413 
Thank you for creating an issue for a feature request.
Sorry, I was busy this weekend.
I will try to implement it tomorrow.

**Comment by koxudaxi** [comment]
@Baruch4413 
I have released the latest version 0.4.9.
If you insert other types then, the library calls `str()` with the passed object.

**Comment by Baruch4413** [comment]
Great!
I updated the package and it works now.
Again, thank you for working on this repository, it is very well constructed and useful.

**Comment by Baruch4413** [comment]
Question:
Would it be possible to return the datetime values as python datetime.datetime objects (Instead of strings)?

I am using Marshmallow to parse and serialize my data, and it treats the data as if it were an instance of datetime.datetime, because it is defined that way in the model.

I think it would be better to return datetime.datetime objects in order to increase compatibility with packages such as Marshmallow.

Peace ✌️

**Comment by Baruch4413** [comment]
I am using SQLAlchemy, this is my current setup:
core.py
```py3
from flask import Flask
from flask_sqlalchemy import SQLAlchemy
app = Flask(__name__)
app.config.from_pyfile('app_config.py')
db = SQLAlchemy(app)
```
app_config.py
```py3
SQLALCHEMY_DATABASE_URI = 'mysql+pydataapi://'
SQLALCHEMY_ENGINE_OPTIONS = {
    'connect_args': {
        'resource_arn': 'arn:aws:rds:us-east-1:123456789012:cluster:cluster-name',
        'secret_arn': 'arn:aws:secretsmanager:us-east-1:123456789012:secret:secret-key',
        'database': 'dbname'
    }
}
```

**Comment by koxudaxi** [comment]
@Baruch4413 
Sorry, I have not used flask_sqlalchemy.
I must search how to return python `datetime.datetime`  from SQLAlchemy :eyes: 
`colspecs` may be to return the map :thinking: 
https://docs.sqlalchemy.org/en/13/core/internals.html#sqlalchemy.engine.interfaces.Dialect

**Comment by Baruch4413** [comment]
flask_sqlalchemy is like SQLAlchemy, it just adds a few methods for pagination and such.
And I don't feel qualified to give my opinion here, but your idea sounds good 👍

**Comment by koxudaxi** [comment]
@Baruch4413 
I just released new version as 0.4.12.
The version returns a value of datetime column as `datetime.datetime` .
